openvpn/start.sh

```diff
@@ -56,7 +56,16 @@ then
 elif [[ "${OPENVPN_PROVIDER^^}" = "FREEVPN" ]]
 then
     FREEVPN_DOMAIN=${OPENVPN_CONFIG%%-*}
-    export OPENVPN_PASSWORD=$(curl -s https://freevpn.${FREEVPN_DOMAIN:-"be"}/accounts/ | grep Password |  sed s/"^.*Password\:.... "/""/g | sed s/"<.*"/""/g)
+    export OPENVPN_PASSWORD=$(curl -s https://freevpn.${FREEVPN_DOMAIN:-"me"}/accounts/ | grep Password |  sed s/"^.*Password\:.... "/""/g | sed s/"<.*"/""/g)
+    
+    # Update FreeVPN certs
+    /etc/openvpn/updateFreeVPN.sh
+    
+    # Add daily update to cron
+    crontab -l > tempcron
+    echo "0 1 * * * /etc/openvpn/updateFreeVPN.sh" > tempcron
+    crontab tempcron
+    rm tempcron
```

Comment:
The configs are set on container/OpenVPN startup, and this script is run on each start of the container.
Am I missing something, or will the cron job not make any difference here? 🤔

Reply:
Hi, the cron is used to automatically call the update script every day at 1:00 AM.
So, this will catch the bundle changes when the container is running for many days.
I'm not sure about container persistency of crontab file, I can be wrong.
Eventually, a check can be added to avoid adding many times the row in crontab.

Reply:
Yeah. But let's say that the container runs for a couple of weeks and then there's a change in the configs. Your script will fetch it and download the file and then what? It won't be reloaded while running, right?

Reply:
Yes, every time the updateFreeVPN.sh script is executed the downloaded bundle files will be put under /etc/openvpn/freevpn , even if the downloaded file are the same of those already present into the container.
There is no reload, actually I've no idea on how to do it.

Reply:
I don't think it's possible. I think OpenVPN must be restarted.
But that will happen automatically anyways. If you run with the --ping-exit options then the old server will not respond anymore (at some point) and the ping will fail and container will exit.
Then with the Docker restart flag (either "always" or "unless-stopped") the container will be restarted and then the start.sh will be triggered again and the bundle will be downloaded then.

So I don't think we need to add it to crontab. I don't think it will help.

Reply:
ah ok, it can be the right way, I didn't know the ping-exit option.

meanwhile I've changed it to add the rule only one time, I'll remove all crontab soon.

Reply:
Removed the crontab changes.